di-ipv-evcs-stub/openAPI/evcs-external.yaml

```diff
@@ -58,6 +58,7 @@ paths:
           $ref: '#/components/responses/BadRequest'
         500:
           $ref: '#/components/responses/ServerError'
+      x-amazon-apigateway-request-validator: ALL
```

Comment:
(Presumably this means we could remove our in-code validation logic - but no urgency)

Reply:
Sure will test it and remove it from code.